### ISSUE TITLE
html lang attribute is missing

### ISSUE THREAD

**Issue by goyalvarun601** [tinymce/tinymce#3580]
I was running a accessibility audit and found this violation.
Can we add the attribute?

**Comment by fyrkant** [comment]
Could you give some more information, exactly where is the attribute missing?

**Comment by goyalvarun601** [comment]
In the html tag like `<html lang="en">`

**Comment by asit-baran-das** [comment]
Any update on this one? I still get this accessibility error in 4.8.5

**Comment by asit-baran-das** [comment]
An workaround would be to add the lang attribute at runtime. 

1. Add this config to tinymce.init = **init_instance_callback: "addLangForAccessibility",**
2. get all elements which has "ifr" string in their ids, as the inline HTML would be inside of <iframe> and then add the attribute to html tag
**function addLangForAccessibility(inst) {
        $('*[id*=ifr]').each(function () {
            var iframe_contents = $(this).contents();
            iframe_contents.find('html').attr("lang", "en");
        });
    }**

**Comment by piltsen** [comment]
I also need the html lang attribute to be set inside thee iframe. Any progress with a possible solution? 

I did try the solution from @asit-baran-das but the problem is that when I try to get the contents (iframe.querySelector('html') - it is always null, because it seems that the editor hasn't initialised it yet. 

1. I added to the init = "init_instance_callback: addLangForAccessibility,"
2. Added the following code
```
const addLangForAccessibility = (editor) => {
  const iframe = document.querySelector('#tinymce_ifr'); // selects ok
  const mainHtml = iframe.querySelector('html') // comes null
}
```

Any help is very much appreciated.

**Comment by piltsen** [comment]
After exploring the issue a bit more I found that the following code works. It might not hold as a solution if you have multiple editors on the screen, but since I only have one, it might do the trick. Not sure if the inner structure changes, it might break this solution. 

```
const addLangForAccessibility = (editor) => {
  editor.contentAreaContainer.children[0].contentDocument.children[0].setAttribute('lang','pt-BR') 
}
```
The second children[0] is the html tag

Hope this helps. 

> I also need the html lang attribute to be set inside thee iframe. Any progress with a possible solution?
> 
> I did try the solution from @asit-baran-das but the problem is that when I try to get the contents (iframe.querySelector('html') - it is always null, because it seems that the editor hasn't initialised it yet.
> 
> 1. I added to the init = "init_instance_callback: addLangForAccessibility,"
> 2. Added the following code
> 
> ```
> const addLangForAccessibility = (editor) => {
>   const iframe = document.querySelector('#tinymce_ifr'); // selects ok
>   const mainHtml = iframe.querySelector('html') // comes null
> }
> ```
> 
> Any help is very much appreciated.

**Comment by vnc5** [comment]
For TinyMCE 6:
```js
init_instance_callback(editor) {
    editor.getDoc().documentElement.lang = 'de'
}
```
